examples/experimental/dagster-airlift/dagster_airlift/in_airflow/base_proxy_operator.py

```diff
@@ -72,6 +89,95 @@ def get_valid_graphql_response(self, response: Response, key: str) -> Any:
 
         return response_json["data"][key]
 
+    @property
+    def full_dag_override(self) -> bool:
+        return False
+
+    def launch_runs_for_dag(self, context: Context, dag_id: str) -> None:
```

Comment:
all this cruft can go away. We can factor out a small "get_targeted_assets" fxn, everything else is shared impl

Reply:
Just in terms of code cleanliness encapsulated all the graphql response shuffling into standalone functions makes sense. (This would be be a good standalone PR to do to the existing code early in the stack)

Reply:
took a pass at this as part of another rfc: https://github.com/dagster-io/dagster/pull/25034